Gemfile.lock

```diff
@@ -0,0 +1,30 @@
+GEM
```

Comment:
You should remove the Gemfile.lock from git

Reply:
👍 Thank you; Not sure how that snuck by me on this PR. I've removed it as of the latest commit.